docs/autogenerated-menu/Our Practices/practices-introduction.md

```diff
@@ -0,0 +1,12 @@
+---
+sidebar_position: 1
+---
+# Practices Introduction
+
+Our company has various core Communities of Practice (CoPs), which are groups of people sharing common interests or knowledge domain. They meet regularly to learn from one another, share good practices, and collaborate on projects. CoPs shape our Products and Services by defining and refining our Service Catalogue and the skills needed to deliver them, ensuring they meet customer requirements whilst fostering innovation and growth as new tools & techniques emerge.
```

Comment:
This feels confusing as internally we no longer have CoPs just Practices.  I think it would make more sense to keep that naming consistent?